ton_sdk/Cargo.toml

```diff
@@ -3,11 +3,11 @@ name = "ton_sdk"
 version = "0.1.0"
 
 [dependencies]
-ton_abi = { git = "ssh://git@github.com/tonlabs/ton-labs-abi.git", branch = "private-repos" }
+ton_abi = { git = "ssh://git@github.com/tonlabs/ton-labs-abi.git", branch = "private-repos-2" } # Fix branch before merge!!!
 ton_block = { git = "ssh://git@github.com/tonlabs/ton-block.git" }
 ton_vm = { git = "ssh://git@github.com/tonlabs/ton-vm.git", default-features = false }
 ton_types = { git = "ssh://git@github.com/tonlabs/ton-types.git" }
-ton_executor = { git = "ssh://git@github.com/tonlabs/ton-executor.git" }
+ton_executor = { git = "ssh://git@github.com/tonlabs/ton-executor.git", branch = "feuture-failure-errors" } # Fix branch before merge!!!
```

Comment:
`error-chain` dependency should be deleted I think

Reply:
Done